api/src/domain/updateSidebarCards.test.ts

```diff
@@ -125,6 +125,86 @@ describe("updateRoadmapSidebarCards", () => {
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).not.toContain("welcome");
+    });
+
+    it("doesn't remove the generic welcome card when operating phase is not UP_AND_RUNNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "FORMED_AND_REGISTERED",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome");
+    });
+
+    it("adds the welcome-up-and-running card when operating phase is UP_AND_RUNNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome-up-and-running");
+    });
+
+    it("removes the generic welcome card when operating phase is UP_AND_RUNNING_OWNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).not.toContain("welcome");
+    });
+
+    it("doesn't remove the generic welcome card when operating phase is not UP_AND_RUNNING_OWNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "FORMED_AND_REGISTERED",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome");
+    });
+
+    it("adds the welcome-up-and-running card when operating phase is UP_AND_RUNNING_OWNING", () => {
+      const userData = generateUserData({
+        profileData: generateProfileData({
+          operatingPhase: "UP_AND_RUNNING_OWNING",
+          industryId: "generic",
+        }),
+        preferences: generatePreferences({ visibleSidebarCards: ["welcome"] }),
+      });
+
+      const updatedUserData = updateSidebarCards(userData);
+      expect(updatedUserData.preferences.visibleSidebarCards).toContain("welcome-up-and-running");
```

Comment:
this test could be combined with "removes the generic welcome card when operating phase is UP_AND_RUNNING_OWNING" - they have the exact same setup and are asserting the same scenario. just one extra assertion statement